scripts/components/LviFavorites.ts

```diff
@@ -21,11 +22,35 @@ export default class LviFavorites extends LviFavoritesDesign {
     set itemTitle(value: string) {
         this.lblFavoriteItemTitle.text = value;
     }
-    get itemPrice(): any {
-        return this.lblFavoriteItemPrice.text;
+    get itemPrice(): string {
+        return this.tvPrice.text;
+    }
+    set itemPrice(value: string) {
+        const discountExists = !!this.tvDiscount.text;
+        let attributeString = new AttributedString({
```

Comment:
We should use const if the variable will not change in the next lines.

Reply:
DONE.